### ISSUE TITLE
Training time [Help needed]

### ISSUE THREAD

**Issue by imransalam** [google-research/pegasus#75]
I want to finetune the pegasus model for a custom dataset. I do not own a GPU and hence have to train it on the cloud (a Tesla K80). Given my dataset has 1000 to 1500 examples (train set). How long does the fine-tuning will take? I am taking the idea that my training will be complete at a reasonable loss output. Thanks.

**Comment by JingqingZ** [comment]
Hi, we didn't try K80 to fine-tune so I am afraid I can't provide an estimation on the time needed. The fine-tuning is supposed to be quick. You may refer to the appendix C of our [paper](https://arxiv.org/pdf/1912.08777.pdf) regarding the steps (or epochs) needed for fine-tuning. The loss output highly depends on the dataset itself and input/target length.

**Comment by cindycandy** [comment]
Hi, I've fine-tuned the aeslc model for 18hours, but it seems that it will take even more hours. As you say, the fine-tuning is supposed to be quick, so I wonder if there is any problem about my operation? But I followed all the steps on README.

**Comment by iIsunnyIi** [comment]
Please share Any custom dataset for Pegasus training.
